bash_completion

```diff
@@ -113,32 +115,38 @@ _sysvdirs()
     return 0
 }
 
+_comp_deprecate_func _sysvdirs _comp_sysvdirs
+
 # This function checks whether we have a given program on the system.
 #
-_have()
+_comp_has_command()
```

Comment:
I would personally go with `_comp_have_command` for this -- the "has" makes it read like asking if completion has a command which is not what we're "asking" with this. `_comp_have_command` sounds just less odd to me. But not a biggie/blocker, leaving up to you.

Reply:
Thank you for the comment. I have switched to `_comp_have_command`